### DISCUSSION TITLE
## Problem: EssentialsX overrides vanilla commands/Spigot's commands/other plugins' commands

### DISCUSSION THREAD

**Discussion by TKDKid1000** [EssentialsX/Essentials#3996] [General]
## Problem: EssentialsX overrides vanilla commands/Spigot's commands/other plugins' commands

**Details:** After installing EssentialsX, commands like `/give` and `/help` are overridden by EssentialsX alternatives.

**Cause:** EssentialsX provides enhanced alternatives to several default commands. For example, EssentialsX's `/give` supports a user-friendly item meta syntax that's less awkward to use than vanilla NBT, while EssentialsX's `/help` allows server owners to create custom help command pages and even customise them per-group. 

**Solution:** See the [commands list](https://essinfo.xeya.me/commands.html) for documentation on how to use EssentialsX's commands.

If you're sure you want to override commands, you can use Bukkit's `commands.yml`. The example below does the following:
* Overrides `/gc` with [spark](https://github.com/lucko/spark)'s `/profiler` command
* Overrides `/tell` with the [vanilla `/tell`](https://minecraft.gamepedia.com/Commands/tell) command
* Forces `/msg` to run EssentialsX's `/msg` command

```yaml
aliases:
    gc:
    - "spark:profiler $1-"
    tell:
    - "minecraft:tell $1-"
    msg:
    - "essentials:msg $1-"
```

_Originally posted by @mdcfe in https://github.com/EssentialsX/Essentials/issues/3956#issuecomment-771707050_

You know that you can simply do /minecraft:cmd when you rarely need to use a vanilla command.

**Comment by mdcfe** [comment]
Yes, we know. This is advice for people who don't want EssentialsX or other plugins to override commands.

In the future, please do not abuse the issue tracker for this. The issue tracker is solely for reporting bugs using the bug report template.
